test/spec/windshaft/client.spec.js

```diff
@@ -22,7 +22,7 @@ describe('windshaft/client', function () {
     });
 
     it('should trigger a GET request to instantiate a map', function () {
-      this.client.instantiateMap({
+      this.client._instantiateMap({
```

Comment:
Does this really make a difference? Why not keep it so that we test the public interface of this object?